### PR TITLE
Use Episode Title when available on TV pages

### LINKED ISSUE #8340: Use Episode Title where available on OD TV pages
## Is your feature request related to a problem? Please describe.
ARES now returns an `episodeTitle` when it has been given by editorial. See `content.blocks.episodeTitle` here: https://ares-api.api.bbci.co.uk/api/ws-media/tv/afrique/bbc_afrique_tv/w3ct05n6

This field will not always be available, but when it **is**, it should be used for:

1. On the page:
- A new `episodeTitle` should be rendered below the current `<BrandTitle/>` [styles here](https://app.zeplin.io/project/5d4d77b1fcbfc79bf47e0294/screen/5f9afa7f7067fe4bd3a531f2)
- The `<StyledParagraphComponent/>` should use `content.blocks. synopses.medium` for its `text` value.

2. In the metadata:
- The metadata html/seo title should be: `{brandTitle} - {episodeTitle} - {serviceName}`

Designs are available for OD TV when this field is available, here https://app.zeplin.io/project/5d4d77b1fcbfc79bf47e0294/screen/5f9afa7f7067fe4bd3a531f2

**Note - confirm with testers that behaviour is correct before releasing**

### Acceptance Criteria
On TV episode and brand pages for the main episode display: 
- The episode media with image placeholder 
- The brand title (e.g BBC Info) 
- The episode title .e.g. "Coronavirus"  
- Medium Synopsis as the paragraph text 
- Release date as a datestamp 

The html / seo page title includes the title where it exists 
- `<Brand title> - <Episode Title> - <Service Name>`
- e.g. "Parlons Covid - le chaud - BBC News Afrique" 

### Scenarios - rendering

**Scenario: Include episode title as part of the episode hero**

Given an `episodeTitle` is present in the ares feed ie: https://ares-api.api.bbci.co.uk/api/ws-media/tv/afrique/bbc_afrique_tv/w3ct05n6
when I visit the tv page https://www.bbc.com/afrique/bbc_afrique_tv/tv/w3ct05n6
then the `episodeTitle` is displayed where the release date usually would be 
and the release date is displayed as a datestamp below the paragraph text (see designs) 

 
**Scenario: Use medium synopsis**

Given an ares response for an episode or brand has a medium synopsis ie https://ares-api.api.bbci.co.uk/api/ws-media/tv/afrique/bbc_afrique_tv/w3ct05n6
when I visit the tv page https://www.bbc.com/afrique/bbc_afrique_tv/tv/w3ct05n6
then the paragraph text displays the medium synopsis 

### Scenarios - SEO/metadata

**Scenario: Include episode title in the html / SEO title**

Given an `episodeTitle` is present in the ares feed https://ares-api.api.bbci.co.uk/api/ws-media/tv/afrique/bbc_afrique_tv/w3ct05n6
and the `episodeTitle` is "Le journal télévisé international de BBC Afrique" 
when I visit the tv page https://www.bbc.com/afrique/bbc_afrique_tv/tv/w3ct05n6
then the SEO / html title of the page will be "BBC Info - Le journal télévisé international de BBC Afrique - BBC News Afrique"

**Given an episode title is NOT present in the ares feed**

when I visit the tv page https://www.bbc.com/afrique/bbc_afrique_tv/tv/w3ct05n6
then the SEO / html title of the page will be "BBC Info - BBC News Afrique" 

### a11y acceptance criteria: 

Screen reader UX: https://zpl.io/b6oKMmy

**Scenario: Hidden H1**

**Scenario: episode title is present**

Given I use a screen reader and the **Episode Title is displayed** 
And I am on a TV page
When I navigate to the main page container
Then H1 is announced first before any other content 
and the H1 is announced as [Brand Title, **Episode Title**] _<- conditional_
and that is followed by the play button on the player (unless iframes are acknowledged) 
and that is then followed by the summary text 
and the visual H1 is NOT acknowledged again

**Scenario: episode title is NOT present**

Given I use a screen reader and the Episode Title is **not** displayed 
And I am on a TV page
When I navigate to the main page container
Then H1 is announced first before any other content 
and the H1 is announced as [Brand Title, **release date**] _<- conditional_
and that is followed by the play button on the player (unless iframes are acknowledged) 
and that is then followed by the summary text 
and the visual H1 is NOT acknowledged again

**Scenario: Video iframe (unchanged)**

Given I use a screen reader
And the I am on a TV page with a video clip
When I navigate to the outer iframe of the SMP on the TV page
And iframe semantics are announced
Then “Video Player” is announced with the semantics

### PR DESCRIPTION
**Pull request by pharingee**
Resolves #8340

**Overall change:**
Change tv description and subtitle when episode  title is available

**Code changes:**

- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/OnDemandHeading/index.jsx [hunk 1]
```diff
@@ -74,14 +76,15 @@ const OnDemandHeadingContainer = ({
 OnDemandHeadingContainer.propTypes = {
   idAttr: string,
   brandTitle: string.isRequired,
-  releaseDateTimeStamp: number.isRequired,
+  releaseDateTimeStamp: number,
```
Comment (rebeccamcginn) [inline src/app/containers/OnDemandHeading/index.jsx]: I know this is still a draft but there should always be a releaseDateTimestamp. If you have found fixture data that does not contain the releaseDateTimestamp, it needs updated with the live Ares response

File: src/app/pages/OnDemandTvPage/OnDemandTvPage.jsx [hunk 2]
```diff
@@ -77,11 +78,18 @@ const OnDemandTvPage = ({ pageData, mediaIsAvailable, MediaError }) => {
     thumbnailImageUrl,
     durationISO8601,
     recentEpisodes,
+    episodeTitle,
+    mediumSynopsis,
```
Comment (ryanmccombe) [inline src/app/pages/OnDemandTvPage/OnDemandTvPage.jsx:82]: We could add these two to the prop types

File: src/app/routes/onDemandTV/getInitialData/index.test.js [hunk 3]
```diff
@@ -118,6 +118,7 @@ describe('Get initial data for on demand tv', () => {
       ['promo', 'media', 'imageUrl'],
       ['promo', 'brand', 'title'],
       ['content', 'blocks', 0, 'imageUrl'],
+      ['content', 'blocks', 0, 'episodeTitle'],
```
Comment (ryanmccombe) [inline src/app/routes/onDemandTV/getInitialData/index.test.js:122]: mediumSynopsis could be added here too

**Review by ryanmccombe — APPROVED** [review]
👍

**Review by RichardPK — APPROVED** [review]

**Pull request merged**